### ISSUE TITLE
lager_syslog rebar.config requires lager 1.x or 2.x, but pulls from lager master

### ISSUE THREAD

**Issue by kennethlakin** [basho/lager_syslog#20]
The lager master branch currently contains lager 3.x.

lager_syslog requires lager version 1.x or 2.x, but pulls from lager master branch. This prevents one from building lager_syslog from a fresh repo clone. lager_syslog should *probably* pull from the 2.x branch.

I *would* make a PR myself, but I'm not sure which branches you'd like to have populated with the change.

See the following conversation with a bash shell for the repro recipe:

```
$ git clone https://github.com/basho/lager_syslog 2> /dev/null
$ cd lager_syslog
lager_syslog $ rebar g-d
WARN:  /tmp/lager_syslog/deps/lager/src/lager.app.src has version "3.0.1"; requested regex was (2.1|2.0|1.0|1.2).*
==> lager_syslog (get-deps)
WARN:  /tmp/lager_syslog/deps/lager/src/lager.app.src has version "3.0.1"; requested regex was (2.1|2.0|1.0|1.2).*
WARN:  /tmp/lager_syslog/deps/lager/src/lager.app.src has version "3.0.1"; requested regex was (2.1|2.0|1.0|1.2).*
ERROR: Dependency dir /tmp/lager_syslog/deps/lager failed application validation with reason:
{version_mismatch,{"/tmp/lager_syslog/deps/lager/src/lager.app.src",
                   {expected,"(2.1|2.0|1.0|1.2).*"},
                   {has,"3.0.1"}}}.
ERROR: 'get-deps' failed while processing /tmp/lager_syslog: rebar_abort
```

**Comment by mrallen1** [comment]
Thanks for the report. We will get this fixed pretty quickly.

**Comment by mrallen1** [comment]
Merged #21 so closing this. Thanks for the report. Reopen if there's any problems.
